docs/source/resources/package-spec.rst

```diff
@@ -284,86 +283,98 @@ parts:
 * The first part is always the exact name of the package.
 
 * The second part refers to the version and may contain special
-  characters:
+  characters.  See table below.
 
-  * \| means OR.
+* The third part is always the exact build string. When there are
+  3 parts, the second part must be the exact version.
```

Comment:
```suggestion
  three parts, the second part must be the exact version.
```
Best to spell out any numbers from zero to ten, unless you're referring to something that must be typed into the UI as numerical.